[2025-01-27T07:54:18.902+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-27T07:54:18.910+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9851100067_2025-01-27T07-52-52-833528Z.check_if_callerid_is_present_in_calllogs_table scheduled__2025-01-27T07:52:52.833528+00:00 [queued]>
[2025-01-27T07:54:18.914+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9851100067_2025-01-27T07-52-52-833528Z.check_if_callerid_is_present_in_calllogs_table scheduled__2025-01-27T07:52:52.833528+00:00 [queued]>
[2025-01-27T07:54:18.914+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-27T07:54:18.919+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): check_if_callerid_is_present_in_calllogs_table> on 2025-01-27 07:52:52.833528+00:00
[2025-01-27T07:54:18.924+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2844) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-27T07:54:18.925+0000] {standard_task_runner.py:72} INFO - Started process 2845 to run task
[2025-01-27T07:54:18.925+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9851100067_2025-01-27T07-52-52-833528Z', 'check_if_callerid_is_present_in_calllogs_table', 'scheduled__2025-01-27T07:52:52.833528+00:00', '--job-id', '1279', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmphm0hx3x7']
[2025-01-27T07:54:18.926+0000] {standard_task_runner.py:105} INFO - Job 1279: Subtask check_if_callerid_is_present_in_calllogs_table
[2025-01-27T07:54:18.949+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9851100067_2025-01-27T07-52-52-833528Z.check_if_callerid_is_present_in_calllogs_table scheduled__2025-01-27T07:52:52.833528+00:00 [running]> on host 680cb867086b
[2025-01-27T07:54:18.988+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9851100067_2025-01-27T07-52-52-833528Z' AIRFLOW_CTX_TASK_ID='check_if_callerid_is_present_in_calllogs_table' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T07:52:52.833528+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-27T07:52:52.833528+00:00'
[2025-01-27T07:54:18.988+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-27T07:54:18.994+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2025-01-27T07:54:18.998+0000] {sql.py:544} INFO - Running statement: SELECT * FROM call_logs WHERE caller_id = 'demo-scheduled-from-ui-worldlinkfbba7651-e90d-4e52-9440-04ab838b501d' limit 1;, parameters: None
[2025-01-27T07:54:18.999+0000] {sql.py:553} INFO - Rows affected: 1
[2025-01-27T07:54:18.999+0000] {python.py:240} INFO - Done. Returned value was: answered
[2025-01-27T07:54:19.010+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-27T07:54:19.010+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9851100067_2025-01-27T07-52-52-833528Z, task_id=check_if_callerid_is_present_in_calllogs_table, run_id=scheduled__2025-01-27T07:52:52.833528+00:00, execution_date=20250127T075252, start_date=20250127T075418, end_date=20250127T075419
[2025-01-27T07:54:19.058+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-27T07:54:19.071+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-27T07:54:19.072+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
